README.md

```diff
@@ -42,19 +60,67 @@ health-checker [options]
 
 | Option | Description | Default 
 | ------ | ----------- | -------
-| `--port` | The port number on which a TCP connection will be attempted. Specify one or more times. | | 
 | `--listener` |  The IP address and port on which inbound HTTP connections will be accepted. | `0.0.0.0:5000`
+| `--config` | A YAML file containing checks which will be evaluated | `health-checks.yml`
 | `--log-level` | Set the log level to LEVEL. Must be one of: `panic`, `fatal`, `error,` `warning`, `info`, or `debug` | `info` 
 | `--help` | Show the help screen | | 
 | `--version` | Show the program's version | | 
 
-#### Example
+```
+health-checker --listener "0.0.0.0:6000" --config "my-checks.yml" --log-level "warning"
+```
 
-Run a listener on port 6000 that accepts all inbound HTTP connections for any URL. When the request is received,
-attempt to open TCP connections to port 5432 and 3306. If both succeed, return `HTTP 200 OK`. If any fails, return `HTTP
-504 Gateway Not Found`.
+#### Configuration
```

Comment:
You might want to add a sentence in this section that says something along the lines of, "You can use a yaml file to configure health-checker with the following settings:"